src/autogluon/bench/cloud/aws/batch_stack/lambdas/lambda_function.py

```diff
@@ -17,7 +20,7 @@ def submit_batch_job(env: list, job_name: str, job_queue: str, job_definition: s
         jobDefinition=job_definition,
         containerOverrides=container_overrides,
     )
-    print(response)
+    logger.info(response)
```

Comment:
Can we add some explanation here so someone looking at the logs would understand the context of what is being logged?

Reply:
Updated.